packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -47,13 +60,20 @@ export default function ListBoxWithTags(props) {
     ) {
       event.stopPropagation();
       onAddCustomOption(label);
-      refFilter.current.reset();
+      if (refFilter.current) {
+        refFilter.current.reset();
```

Comment:
does it need to be reset because will rerender and get into the wrong state?

Reply:
Yep in the existing code it was already being reset() since if the user selects an option from a filtered list, the filter should be cleared and reset the `filteredOptions` state.